committee-code-of-conduct/election.md

```diff
@@ -45,20 +45,34 @@ Nominations can be collected via any means. In the past a Google form has been u
 
 The nomination form must be communicated via email to [Kubernetes dev](https://groups.google.com/forum/#!forum/kubernetes-dev) mailing list.
 
-Prior to the voting process, Steering must:
-* Vet that all nominees support the role of the code of conduct committee, i.e. maintaining a safe and inclusive space for everyone to work on Kubernetes.
+Prior to the voting process, Steering Committee must:
+* Vet that all nominees support the role of the Code of Conduct Committee, i.e. maintaining a safe and inclusive space for everyone to work on Kubernetes.
 * Confirm with all nominees (that were not self nominations) that they accept the nomination.
 
 ### Maximal representation
 
 Maximal representation of a single employer is `QUORUM - 1`. With a five person committee,
 that limits the number of members employed by the same company to two.
 
+### Committee Conflict of Interest
+
+As project committees have unique charters, there is a possibility
+for a conflict of interest to arise.  While multiple scenarios may
+be possible now or in the future, one known example of such conflict
+of interest is where a member of the Code of Conduct Committee is
+elected to the Steering Committee.
+
+Such conflict will be resolved by the applicable election committee
+reaching out to the newly-elected member and facilitating a decision
+regarding on which committee the individual will continue.  The
+committee where that individual vacated their seat will fill that
+vacancy as per existing election or committee vacancy rules.
```

Comment:
```suggestion
At time of writing, the only known conflict of interest for Code of Conduct Committee 
members and appointees is nomination to the Steering Committee. 
Members should not serve on both committees simultaneously.

Upon nomination, the (Steering Committee) election committee will each out to 
the newly elected Steering Committee member and ask which committee they intend to continue with. 

The committee where that individual vacated their seat will fill that 
vacancy per existing election or committee vacancy rules.
```

Reply:
"Upon nomination, the (Steering Committee) election committee will each out to"
assuming reach instead of each^^:;;

Reply:
yes :D

Reply:
`Members should not serve on both committees simultaneously` reads too weak for me with "should".